documentation/1.3/tour/initialization.md

```diff
@@ -44,7 +44,7 @@ We can resolve the name collision using the keyword `this`.
 
     class Greeting(String name) {
         shared void greet(String name) {
-            print("``name`` says 'Hello ``this.name``!'"); //oops, local name hides this.name!
+            print("``name`` says 'Hello ``this.name``!'");
```

Comment:
This code snippet uses `this.name` to resolve the hiding which was already highlighted in the [previous code snippet](https://github.com/ceylon/ceylon-lang.org/pull/485/files#diff-f4d23fc165d22d17ca702bb546b4450cR39), so seemed the comment in this snippet should have been different to highlight the resolution, or removed. Please ignore if intentional.